artman/cli/main.py

```diff
@@ -113,6 +111,14 @@ def parse_args(*args):
              '/path/to/config.yaml:config_section_A|config_section_B. '
              'This is an advanced feature; for most cases, use `--api`.',
     )
+    api.add_argument('--batch',
```

Comment:
I thought about that approach, but I think having it as a separate flag is clearer, because other CLI behaviour is changed (e.g. changed default pipeline, not accepting the `--target` arg), and having that behaviour change just based on the format of the args to `--api` seems less clear.

Luke's point about opt-in is also a good one.